DependencyInjection/ONGRElasticsearchExtension.php

```diff
@@ -34,6 +36,11 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('services.yml');
+
+        if (Kernel::MAJOR_VERSION >= 4) {
+            $loader->load('services4.yml');
```

Comment:
You could use `services.yaml` here. SF4 changed from `yml` to `yaml` and this can be of use here.